chapters/ch40.md

```diff
@@ -1,104 +1,102 @@
 > # 40. Generalizing from the training set to the dev set
 
---> _replace THIS LINE by your translation for the above line_
+--> ### 40. Tổng quát hóa từ tập huấn luyện đến tập phát triển
 
 
 > Suppose you are applying ML in a setting where the training and the dev/test distributions are different. Say, the training set contains Internet images + Mobile images, and the dev/test sets contain only Mobile images. However, the algorithm is not working well: It has a much higher dev/test set error than you would like. Here are some possibilities of what might be wrong:
 
---> _replace THIS LINE by your translation for the above line_
-
+--> Giả sử bạn đang áp dụng Machine Learing cho một ứng dụng mà phân phối của tập huấn luyện và tập phát triển khác nhau. Tập huấn luyện chứa ảnh Internet + ảnh Điện Thoại trong khi tập phát triển/kiểm tra chỉ chứa ảnh Điện Thoại. Tuy nhiên, thuật toán của bạn không thể hiện tốt: Số lượng lỗi trên tập phát triển/kiểm tra cao hơn nhiều con số mà bạn muốn. Dưới đây là một số khả năng có thể gây ra vấn đề trên:
 
 > 1. It does not do well on the training set. This is the problem of high (avoidable) bias on the training set distribution.
 
---> _replace THIS LINE by your translation for the above line_
+--> Thuật toán không thể hiện tốt trên tập huấn luyện. Vấn đề này gọi là độ chệnh (tránh được) cao trên phân phối của tập huấn luyện.
 
 
 > 2. It does well on the training set, but does not generalize well to previously unseen data *drawn from the same distribution as the training set*.​ This is high variance.
 
---> _replace THIS LINE by your translation for the above line_
+--> Thuật toán thể hiện tốt trên tập huấn luyện, nhưng không tổng quát hóa tốt trên dữ liệu chưa thấy trích xuất *từ cùng phân phối với tập huấn luyện*. Trường hợp này là phương sai cao.
```

Comment:
Thuật toán ~~thể hiện~~hoạt động tốt trên tập huấn luyện

Reply:
từ in nghiêng phải bắt đầu từ "trích xuất".

Reply:
thiếu số "2. "